docs/requirements.txt

```diff
@@ -3,3 +3,7 @@
 # SPDX-License-Identifier: Unlicense
 
 sphinx>=4.0.0
+adafruit-circuitpython-esp32spi>=5.0.0
```

Comment:
Everything but the `sphinx` requirement should get moved to the outermost `requirements.txt`

Reply:
I cannot believe I didn't catch this. I'm sorry, this is such a rookie mistake.